src/util-file.h

```diff
@@ -89,6 +89,10 @@ typedef struct File_ {
                                      *   flag is set */
     uint64_t content_stored;
     uint64_t size;
+
+    uint32_t *sid; /* signature id of a rule that triggered the filestore event */
+    uint32_t sid_sze;
```

Comment:
it's unclear to me what these fields represent? Size in bytes or in elements? How do they relate?